interaction/README.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Drop new line in front of file.

Reply:
Done via https://github.com/bpmn-io/bpmn-js-examples/pull/113/commits/e5b63b15fed9584dc4dcba1f1be64dcfb546c28b